blogs/lokiless/lokiless.md

```diff
@@ -0,0 +1,282 @@
+
+This blog post talks about a new feature that comes with Network Observability (NetObserv) 1.4: the ability to deploy it without Loki, coupled with exporting flows to an alternative storage. We will try to explain what this new feature means to you, users, and to us, NetObserv developers.
+
+## A brief history
+
+When we started the NetObserv project, one of the first architectural questions was, as you could expect, which storage solution to adopt. It has to be robust for a write-intensive application, with indexing capabilities on large data sets, scalable, while still allowing to run complex queries. It must be able to store structured logs and to extract time-series from them. Features like full-text indexing or data mutability aren't required. On top of that, the license must be compatible with our needs. We ended up with a shortlist that included Grafana Loki, OpenDistro, Influx and a few others. This was two years ago.
+
+We also talked with other OpenShift teams having similar requirements, such as the Logging and the Distributed Tracing teams, and got some feedback to eventually rule out candidates with supposedly higher operational costs. And since the Logging team had already planned to invest in the Loki Operator, that was a nice opportunity to mutualize some efforts. Ok, let's be honest: that was a huge time saver especially for us, thanks so much folks!
+
+## Why change now?
+
+To be clear, **we aren't actually moving away from Loki**. Loki remains the one and only storage solution that we fully support at the moment, and our Console plugin is entirely based on queries to Loki, in its _logql_ format. However, we have seen some people using NetObserv in a way that we didn't expect: deploying it without Loki, and configuring flow exporters for instance with Kafka or IPFIX. Why? It turned out they were more interested in the kube-enriched raw flow data than in the visualizations that NetObserv provides, and they didn't want to deal with a new backend storage setup and maintenance. Which, admittedly, is a quite reasonable argument.
+
+To summarize, here's the deal:
+- 💰: you save on operational aspects by not deploying Loki or any storage that would be new to you.
+- 💔: you lose all the fancy dashboards that we build with so much love.
+- 👩🏽‍🔧: you need to create your own consumer — Kafka or IPFIX — for doing anything with the exported flows: this is on your plate. But you will see below that it's nothing insurmountable.
+
+## What is changed
+
+Before 1.4, there was still a problem: NetObserv was designed with Loki as a requirement. If you don't configure the Loki endpoint, our _flowlogs-pipeline_ component still tries to send flows to a default URL (and fails), and our Console plugin still tries to query Loki (and fails). While the latter isn't too annoying for someone who anyway doesn't want to use the Console plugin, the former could be the cause of performance degradation.
```

Comment:
This paragraph is past-tense by nature because it describes the "Before 1.4 NetObserv", so I applied consistent past-tense in this paragraph. There was some mixing of tenses. Also the same capital C console from before.

Reply:
thanks - I'm rewritting it, with parts of your suggestion